cmd/installer/root.go

```diff
@@ -30,9 +30,31 @@ func AddPack(packPath string, checkEula, extractEula bool) error {
 
 	log.Infof("Adding pack \"%s\"", packPath)
 
+	dropPreInstalled := false
+	fullPackPath := ""
+	backupPackPath := ""
 	if !extractEula && pack.isInstalled {
-		log.Infof("pack %s is already installed here: %s", packPath, filepath.Join(Installation.PackRoot, pack.Vendor, pack.Name, pack.GetVersion()))
-		return nil
+		if forceReinstall {
+			log.Infof("Making temporary backup of pack \"%s\"", packPath)
+			// Get target pack's full path and move it to a temporary "_tmp" directory
+			var packInfo utils.PackInfo
```

Comment:
From what I could see the use of `packInfo`, it is just to get Vendor, Pack and Version. If that is the only use of it, you could re-use the `pack` variable, declared on line 26.

```golang
26. pack, err := preparePack(packPath)
```
`pack` contains Vendor, Pack and Version as well